tests/acceptance/00_basics/def.json/server_dirs.cf.json

```diff
@@ -0,0 +1,7 @@
+{
+    "vars": {
+        "server_controls": {
```

Comment:
@basvandervlies @tzz lets bikeshed this name a bit.

0) I feel like I am terrible at picking names 
1) I don't consider this a "server control" because access rules can be in any server bundle.
2) Why nest this deeply instead of `mpf_dir_templates`, `mpf_templates` or `templates`?
3) Why do I suggest MPF anything? Because highlighting the fact that its a MPFism and not a core thing seems like a good thing to do.